### ISSUE TITLE
Accessibility focus freezes when navigating through treemap points

### ISSUE THREAD

**Issue by raf18seb** [highcharts/highcharts#15343]
Like in the title.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/mxzo5rcn/

#### Product version
Highcharts 9.0.1

FYI: @oysteinmoseng
